src/client/datascience/notebookStorage/vscNotebookModel.ts

```diff
@@ -83,15 +83,12 @@ export class VSCodeNotebookModel extends BaseNotebookModel {
         return this.document ? this.document.cells.length : this.notebookJson.cells?.length ?? 0;
     }
     public getNotebookData() {
-        if (!this.preferredLanguage) {
-            throw new Error('Preferred Language not initialized');
```

Comment:
Its possible to not have a preferred langauge, we'll default to plain text.

Reply:
Note: When you create a blank notebook and we return cells with plain text,
& a kernel is selected, then the language of the cells get updated to match the language of the kernel selected.

Tha'ts existing fucntionality.

I.e. if you create a blank notebook and cells are TypeScript, then select Juali kernel, then language of empty cells in blank notebook will change to Julia.